.github/workflows/labeler-module.yml

```diff
@@ -0,0 +1,16 @@
+name: "Pull Request Labeler"
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+
+  triage_file_extensions:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: thomasjpfan/labeler@v2.5.0
```

Comment:
I made this fork to add a [max-labels feature](https://github.com/thomasjpfan/labeler#change-in-this-fork) which puts a upper bound on how many labels the labeler can add. If you do not need this feature, I'll go with https://github.com/actions/labeler.

Reply:
We were using this one but had issues. See these PRs #14677 #14709 and the PR that removed it #14967

Reply:
Ah yes, that is another feature I added to my fork: https://github.com/thomasjpfan/labeler/commit/64aeb548e93828fab1e5590c6b295e5f8ab5d67a . My fork does not label if the PR is already labeled with any of the matching labels. Matching labels are the ones configured in the yaml.

Concretely, with this PR, if someone manually labeled a PR with Cython, then my fork will do nothing. This is because `Cython` is one of the labels in `.github/labeler-file-extensions.yml`.

Reply:
Now that I think about it, that was a time before `pull_request_target`. With `pull_request_target` set to trigger on `opened` or `created`, the label should not sync anymore and run once.

This has the downside of the labels not being in sync, but it will not override any labels. At scikit-learn, we use `opened` and that has been sufficient over the last ~2 years.

Reply:
Ok thanks for the precisions. I believe if we have the insurance there is no sync, people would be ok to reconsider 😃 (still needs some discussion on the mailing list)

Reply:
I just have sent an email on the SciPy-Dev mailing list as you proposed.